ttnn/cpp/ttnn/operations/full/device/full_device_operation.hpp

```diff
@@ -2,6 +2,8 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#pragma once
```

Comment:
I don't believe this... we didn't have `#pragma once` on these headers???

cc: @afuller-TT @TT-billteng @blozano-tt

Reply:
Yes, I was quite surprised as well

Reply:
The funny one was the one that had it... but had it commented out 😂 
It's not about where we were yesterday, it's about where we'll be tomorrow :)